aict_tools/cta_helpers.py

```diff
@@ -0,0 +1,171 @@
+import numpy as np
+import astropy.units as u
+from astropy.coordinates import SkyCoord
+from astropy.coordinates import EarthLocation
+from astropy.coordinates import AltAz
+from astropy.time import Time
+import logging
+import warnings
+
+from astropy.coordinates import SkyCoord, AltAz
+from ctapipe.coordinates import (
+    NominalFrame,
+    CameraFrame,
+    TiltedGroundFrame,
+    project_to_ground,
+    GroundFrame,
+    MissingFrameAttributeWarning
+)
+from ctapipe.instrument import TelescopeDescription
+
+log = logging.getLogger(__name__)
+
+
+def horizontal_to_camera_cta_simtel(df):
```

Comment:
give single arrays as arguments. palt, paz, alt, az, and focal_length

Reply:
(Like this, we can use our parallel array computation function)

Reply:
Thats probably the best option, agreed